sql/moz-fx-data-shared-prod/telemetry/unified_metrics/view.sql

```diff
@@ -2,6 +2,7 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.unified_metrics`
 AS
 SELECT
-  * REPLACE (COALESCE(country, '??') AS country)
+  * REPLACE (COALESCE(country, '??') AS country),
+  `mozfun.norm.browser_version_info`(app_version) as browser_version_info
```

Comment:
I ran `bqetl format sql/moz-fx-data-shared-prod/telemetry/unified_metrics/view.sql` and this was the diff:

```suggestion
  `mozfun.norm.browser_version_info`(app_version) AS browser_version_info
```